### PR TITLE
OCPBUGS-17499: Adding reset step to Booting from an HTTP-hosted ISO image using the Redfish API

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.13, 4.14, 4.15

Issue: https://issues.redhat.com/browse/OCPBUGS-17499

Link to docs preview: https://69874--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-installing-sno#install-booting-from-an-iso-over-http-redfish_install-sno-installing-sno-with-the-assisted-installer

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/install-booting-from-an-iso-over-http-redfish.adoc [hunk 1]
```diff
@@ -53,3 +53,17 @@ $ curl -k -u <bmc_username>:<bmc_password> -d '{"ResetType": "ForceRestart"}' -H
...
+[source,terminal]
+----
+$ curl -k -u <bmc_username>:<bmc_password> -d '{}' -H "Content-Type: application/json" -X POST <host_bmc_address>/redfish/v1/Managers/iDRAC.Embedded.1/VirtualMedia/CD/Actions/VirtualMedia.EjectMedia
+----
+
+. Reset the boot mode by running the following command:
++
+[source,terminal]
+----
+$ curl -k -u <bmc_username>:<bmc_password> -X PATCH -H 'Content-Type: application/json' -d '{"Boot": {"BootSourceOverrideTarget": "None", "BootSourceOverrideMode": "Legacy", "BootSourceOverrideEnabled": "Once"}}' <host_bmc_address>/redfish/v1/Systems/System.Embedded.1
```
Comment (amolnar-rh) [inline modules/install-booting-from-an-iso-over-http-redfish.adoc:68]: Please check if the `"Legacy"` value is correct in `"BootSourceOverrideMode": "Legacy"`

**Comment by dfitzmau** [comment]
Hi @amolnar-rh .

Looks like yopur build has passed. Would you be OK to update the PR with a dummy update? I'll issue a `retest` command to check if that refreshed the build.

Also would you be OK to get an SME and QE ack?

**Comment by amolnar-rh** [comment]
@dfitzmau Got all the explicit acks. Pushed again, but the tests don't seem to be passing

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**